contracts/Tokens/Prime/Prime.sol

```diff
@@ -404,8 +434,11 @@ contract Prime is AccessControlledV8, PausableUpgradeable, MaxLoopsLimitHelper,
     function issue(bool isIrrevocable, address[] calldata users) external {
         _checkAccessAllowed("issue(bool,address[])");
 
+        uint256 usersLength = users.length;
+        _ensureMaxLoops(usersLength);
```

Comment:
I think I wouldn't add this check here, and I would leave the transaction to fail due to out-of-gas